example/src/screens/CustomSubtitleOnlyUI.tsx

```diff
@@ -0,0 +1,243 @@
+  console.log(header, JSON.stringify(obj, null, 2));
+}
+
+interface TrackDisplay {
+  display: string;
+  identifier: string;
+}
+
+const Separator = () => <View style={styles.separator} />;
+
+const trackDisplayKeyExtractor = ({ identifier }: TrackDisplay) => identifier;
+
+export default function CustomSubtitleOnlyUI() {
+  useTVGestures();
+  const [playing, setPlaying] = useState<boolean>(false);
+
+  const styleConfig: StyleConfig =
+    Platform.OS === 'android'
+      ? {
+          isUiEnabled: false,
+        }
+      : {
+          userInterfaceType: UserInterfaceType.subtitle,
+        };
+
+  const player = usePlayer({
+    licenseKey: '60bd09ff-5a03-45ed-9453-e832798d1c7e',
+    styleConfig,
+  });
+  const [tracks, setTracks] = useState<TrackDisplay[]>([]);
+
+  useFocusEffect(
+    useCallback(() => {
+      player.load({
+        url:
+          Platform.OS === 'ios'
+            ? 'https://bitmovin-a.akamaihd.net/content/sintel/hls/playlist.m3u8'
+            : 'https://bitmovin-a.akamaihd.net/content/sintel/sintel.mpd',
+        type: Platform.OS === 'ios' ? SourceType.HLS : SourceType.DASH,
+        poster: 'https://bitmovin-a.akamaihd.net/content/sintel/poster.png',
+      });
+      return () => {
+        player.destroy();
+      };
+    }, [player])
+  );
+
+  const onReady = useCallback(
+    async (event: Event) => {
+      prettyPrint(`EVENT [${event.name}]`, event);
+      const subtitleTracks = await player.getAvailableSubtitles();
+
+      setTracks(
+        subtitleTracks
+          .filter((t) => !!t.identifier && !!t.label)
+          .map(
+            (t): TrackDisplay => ({
+              display: t.label as string,
+              identifier: t.identifier as string,
+            })
+          )
+      );
+    },
+    [player]
+  );
+
+  const onEvent = useCallback((event: Event) => {
+    prettyPrint(`EVENT [${event.name}]`, event);
+  }, []);
+
+  const onPlaying = useCallback(
+    (event: PlayingEvent) => {
+      onEvent(event);
+      setPlaying(true);
+    },
+    [onEvent]
+  );
+
+  const onPaused = useCallback(
+    (event: PausedEvent) => {
+      onEvent(event);
+      setPlaying(false);
+    },
+    [onEvent]
+  );
+
+  const renderTrack = useCallback(
+    ({ item }: ListRenderItemInfo<TrackDisplay>) => {
+      return (
+        <TouchableOpacity
+          style={styles.trackItem}
+          onPress={() => player.setSubtitleTrack(item.identifier)}
+        >
+          <Text style={styles.trackItemText}>{item.display}</Text>
+        </TouchableOpacity>
+      );
+    },
+    [player]
+  );
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.playerContainer}>
+        <PlayerView
+          player={player}
+          style={styles.player}
+          onReady={onReady}
+          onEvent={onEvent}
+          onPlaying={onPlaying}
+          onPaused={onPaused}
+          onAudioChanged={onEvent}
+          onSubtitleChanged={onEvent}
+        />
+        <SubtitleView
```

Comment:
This only renders when the Platform.OS is android, because iOS has it's own setting.